### PR TITLE
safeMsg is not available in persistent scripts

### PR DESCRIPTION
**Pull request by RedGuy12**
```js
    module.default({
      addon: addonObj,
      global: globalObj,
      console: { ...console, log, warn },
      setTimeout: setTimeoutFunc,
      setInterval: setIntervalFunc,
      clearTimeout: clearTimeoutFunc,
      clearInterval: clearIntervalFunc,
      msg,
    });
```

no `safeMsg`!

### CODE REVIEW

**Comment by RedGuy12** [comment]
wait wut the commit didn't work

**Comment by RedGuy12** [comment]
finally! it kept bugging out

**Comment by Hans5958** [comment]
cc @WorldLanguages

File: content/docs/localization/localizing-addons.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ hungry2 = msg("eat", {food: msg("salmon")}) // shows "I want to eat salmon!"
 ```
 
 ### Safety
-If you are writing raw HTML, `msg` should be replaced with safer version of `safeMsg`.
+If you are writing raw HTML, `msg` should be replaced with safer version of `safeMsg`. Please note that `safeMsg` is not available in persistent scripts.
 
 ## Enabling localization
-Edit your `addon.json` to add `"l10n": true`. That's all you have to do!
\ No newline at end of file
+Edit your `addon.json` to add `"l10n": true`. That's all you have to do!
```
Comment (WorldLanguages) [inline content/docs/localization/localizing-addons.md:56]: Now that we're on it, this line (and paragraph title) can be removed

Comment (RedGuy12) [reply content/docs/localization/localizing-addons.md]: done

**Pull request merged**